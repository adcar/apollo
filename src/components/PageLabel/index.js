import React from 'react'
import { withStyles } from '@material-ui/core/styles'
import Typography from '@material-ui/core/Typography'

const styles = theme => ({
	root: {
		marginTop: theme.spacing.unit * 2,
		marginBottom: theme.spacing.unit * 2
	}
})

const PageLabel = ({ classes, children }) => {
	return (
		<Typography
			className={classes.root}
			variant="display1"
			component="h1"
			align="center"
		>
			{children}
		</Typography>
	)
}

export default withStyles(styles)(PageLabel)
