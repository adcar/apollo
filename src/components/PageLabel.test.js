import React from 'react'
import PageLabel from './PageLabel'
import { shallow, mount, render } from 'enzyme'
import toJson from 'enzyme-to-json'
test('CardWrapper renders correctly', () => {
	const wrapper = shallow(<PageLabel>Test Page!</PageLabel>)
	expect(toJson(wrapper)).toMatchSnapshot()
})
