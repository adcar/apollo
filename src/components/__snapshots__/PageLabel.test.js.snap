// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardWrapper renders correctly 1`] = `
<PageLabel
  classes={
    Object {
      "root": "PageLabel-root-1",
    }
  }
>
  Test Page!
</PageLabel>
`;
